src/index.js

```diff
@@ -0,0 +1,72 @@
+import './styles/style.css';
+
+const submit = document.querySelector('.submit');
+const refresh = document.querySelector('.refresh');
+const sendScores = document.querySelector('.score-record');
+const name = document.getElementById('name');
+const score = document.getElementById('score');
+
+class Scores {
+  static scoreList = [];
+
+  constructor(user, score) {
+    this.user = user;
+    this.score = score;
+  }
+}
+
+let uniqueIdentifier = '';
+const gamer = async () => {
+  await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games',
+    {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        name: 'my cool new game',
+      }),
+    },
+  ).then((res) => res.json().then((data) => {
+    uniqueIdentifier = data.result;
+  }));
+};
+gamer();
+
+submit.addEventListener('click', async () => {
+  const newScore = new Scores(name.value, score.value);
+  await fetch(
+    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${uniqueIdentifier}/scores/`,
```

Comment:
- [ ] When the submit button is clicked, data is not successively posted to the Leaderboard API. It seems you didn't generate a unique id for POST and GET actions. Please kindly fix this issue.

Reply:
https://www.loom.com/share/8f591595b42f44aeae57bd4d3fbe7d76